src/main/java/io/github/moulberry/notenoughupdates/miscgui/GuiCustomEnchant.java

```diff
@@ -606,15 +606,15 @@ public void render(float partialTicks) {
                     Gui.drawRect(guiLeft + 295, guiTop + 147, guiLeft + 295 + 16, guiTop + 147 + 16, 0x80ffffff);
                     tooltipToDisplay = createTooltip("Enable GUI", 0, "On", "Off");
                     break;
-                case 1:
-                    Gui.drawRect(guiLeft + 295 + 18, guiTop + 147, guiLeft + 295 + 16 + 18, guiTop + 147 + 16, 0x80ffffff);
-                    tooltipToDisplay = createTooltip("Incompatible Enchants",
-                            NotEnoughUpdates.INSTANCE.config.enchantingSolvers.incompatibleEnchants,
-                            "Highlight", "Hide");
-                    tooltipToDisplay.add(1, EnumChatFormatting.GRAY + "How to display enchants that are");
-                    tooltipToDisplay.add(2, EnumChatFormatting.GRAY + "incompatible with your current item,");
-                    tooltipToDisplay.add(3, EnumChatFormatting.GRAY + "eg. Smite on a sword with Sharpness");
-                    break;
+                //case 1:
```

Comment:
please use a block comment

Reply:
I tried here and in the next spot and IntelliJ wasnt happy so I had to do it like this

Reply:
or maybe im just pepega :4Shrug: